tools/tpp-run/CMakeLists.txt

```diff
@@ -64,7 +60,6 @@ target_link_options(tpp-run PRIVATE
   -lmlir_async_runtime
   ${TPP_GPU_LINK_FLAGS}
   -lomp
-  -Wl,--as-needed
```

Comment:
This was there to close the `no-as-needed` above. It is necessary if something adds more deps after this.